app/controllers/JavaListController.scala

```diff
@@ -72,6 +72,7 @@ class JavaListController @Inject() (versionRepo: VersionsRepository, cc: Control
     "amzn"    -> "Amazon",
     "grl"     -> "GraalVM",
     "librca"  -> "BellSoft",
+    "nik"     -> "NativeImageKit",
```

Comment:
Just a thought, but how about calling it `Bellsoft NIK` or `Liberika NIK`?

Reply:
agree! just though there is a limitation of string length

Reply:
`NativeImageKit` is two characters longer than `Liberica NIK`, so that should be okay, right?

Reply:
updated!